PyRoute/Star.py

```diff
@@ -42,7 +42,7 @@ def __init__ (self):
                        'Baronets': 0,
                        'Barons': 0,
                        'Marquis': 0,
-                       'Vicounts': 0,
+                       'Viscounts': 0,
```

Comment:
Take a look at the history of Tests/TestNobles.py to see how I verified this change.